networkx/algorithms/planarity.py

```diff
@@ -24,8 +25,29 @@ def check_planarity(G, counterexample=False):
         If the graph is planar `certificate` is a PlanarEmbedding
         otherwise it is a Kuratowski subgraph.
 
+    Examples
+    --------
+
+    This function can be considered as the primary interface for the planar embedding tools since
+    -if the graph is planar- it returns a PlanarEmbedding which can be then used to call other methods of this class.
+    Consider the following example:
+
```

Comment:
Again more nits :) This is entirely subjective, but IMO the example is just as good without the exposition here. In other words I think this text can safely be removed and let the executable part of the example speak for itself.

I realize the wording was probably intended to address this comment:

> Let's update the documentation to make it clear that the check_planarity function is the primary interface for the planar embedding tools. 

from the linked issue, but I think this might be better addressed by mentioning `check_planarity` in the `PlanarEmbedding` docstring as you've already done (see comment below for related suggestions).